# Respostas

1)

A) Quantitativa, da para aplicar operações e continua, pois pode estar aproximado.

B) Quantitativa, da para aplicar operações e continua, pois pode esta aproximado.

C) Qualitativa, pois descrevem caracteristicas.

D) Qualitativas, pois descrevem caracteristicas e não da para fazer operações

E) Depende, as vezes qualitativas, as vezes quantitativas.

2)

a) Sexo, Predileta = qualitativas, Nota = quantitativa, discreta

b)

In [3]:
import pandas as pd
import numpy as np 

In [4]:
aulaCalouro = {
'Sexo' : ['F','M','F','F','M','M','F','F','F','M','F','M','M','M','M','M','F','F','M','M','F','F','M','M','F','F','M','M','F','F','F','M','M','F','M','M','F','M','F','F','F','M'],
'Predileta' : ['H','M','P','H','C','H','M','P','H','C','P','M','G','G','M','M','P','G','G','P','M','G','H','M','P','G','C','H','M','P','M','G','H','M','P','G','M','P','M','M','P','M'], 
'Nota' : [5,8,8,6,5,6,8,4,2,6,8,2,4,9,7,1,8,5,9,5,8,6,3,5,5,5,7,4,7,7,6,6,9,8,5,6,7,5,5,5,9,8]}

In [5]:
df = pd.DataFrame(aulaCalouro)
df

Sexo Predileta  Nota
0     F         H     5
1     M         M     8
2     F         P     8
3     F         H     6
4     M         C     5
5     M         H     6
6     F         M     8
7     F         P     4
8     F         H     2
9     M         C     6
10    F         P     8
11    M         M     2
12    M         G     4
13    M         G     9
14    M         M     7
15    M         M     1
16    F         P     8
17    F         G     5
18    M         G     9
19    M         P     5
20    F         M     8
21    F         G     6
22    M         H     3
23    M         M     5
24    F         P     5
25    F         G     5
26    M         C     7
27    M         H     4
28    F         M     7
29    F         P     7
30    F         M     6
31    M         G     6
32    M         H     9
33    F         M     8
34    M         P     5
35    M         G     6
36    F         M     7
37    M         P     5
38    F         M     5
39    F         M     5
40    F         P     9
41    M         M     8

In [6]:
df['Sexo'].value_counts()

M    21
F    21
Name: Sexo, dtype: int64

In [7]:
df['Sexo'].unique()

array(['F', 'M'], dtype=object)

In [9]:
df['Sexo'].value_counts(normalize=True)*100

M    50.0
F    50.0
Name: Sexo, dtype: float64

50% masculino, 50% feminino

In [10]:
df['Predileta'].value_counts()

M    14
P    10
G     8
H     7
C     3
Name: Predileta, dtype: int64

In [11]:
df['Predileta'].value_counts(normalize=True)*100

M    33.333333
P    23.809524
G    19.047619
H    16.666667
C     7.142857
Name: Predileta, dtype: float64

33.3% matematica, 23.8% portugues, 19.04% geografia, 16.66% historia, 7.14%  ciencias

3)

A) Qualitativa = Pap, Gi, Quantitativa = Res, Renda

B)

In [20]:
dadosFamilia = {
    'Ident':[1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40],
    'PAP':[0,0,1,0,1,1,0,1,1,1,0,1,0,0,0,0,1,0,0,1,1,1,0,0,0,1,1,0,1,0,1,1,1,1,1,1,1,1,1,1],
    'GI':[3,3,2,2,3,1,3,3,3,2,2,1,2,2,3,2,3,3,3,3,3,3,3,2,2,3,1,3,2,3,3,2,3,1,1,1,3,3,3,3],
    'RES':[4,4,4,5,4,1,2,3,6,4,6,4,4,6,3,5,3,4,4,4,5,5,5,4,8,4,5,3,4,5,5,2,5,2,6,8,3,5,5,3],
    'Renda':[10.3,15.4,9.6,5.5,9,2.4,4.1,8.4,10.3,4.6,18.6,7.1,12.9,8.4,19.3,10.4,8.9,12.9,5.1,12.2,5.9,12.9,7.7,1.1,7.5,5.8,7.2,8.6,5.1,2.6,7.7,2.4,4.8,2.1,4,12.5,6.8,3.9,9,10.9]
}


In [21]:
df = pd.DataFrame(dadosFamilia)
df

Ident  PAP  GI  RES  Renda
0       1    0   3    4   10.3
1       2    0   3    4   15.4
2       3    1   2    4    9.6
3       4    0   2    5    5.5
4       5    1   3    4    9.0
5       6    1   1    1    2.4
6       7    0   3    2    4.1
7       8    1   3    3    8.4
8       9    1   3    6   10.3
9      10    1   2    4    4.6
10     11    0   2    6   18.6
11     12    1   1    4    7.1
12     13    0   2    4   12.9
13     14    0   2    6    8.4
14     15    0   3    3   19.3
15     16    0   2    5   10.4
16     17    1   3    3    8.9
17     18    0   3    4   12.9
18     19    0   3    4    5.1
19     20    1   3    4   12.2
20     21    1   3    5    5.9
21     22    1   3    5   12.9
22     23    0   3    5    7.7
23     24    0   2    4    1.1
24     25    0   2    8    7.5
25     26    1   3    4    5.8
26     27    1   1    5    7.2
27     28    0   3    3    8.6
28     29    1   2    4    5.1
29     30    0   3    5    2.6
30     31    1   3    5    7.7
31     32    1   2    2    2.4
32     33    1   3    5    4.8
33     34    1   1    2    2.1
34     35    1   1    6    4.0
35     36    1   1    8   12.5
36     37    1   3    3    6.8
37     38    1   3    5    3.9
38     39    1   3    5    9.0
39     40    1   3    3   10.9

In [22]:
df['PAP'].value_counts()

1    24
0    16
Name: PAP, dtype: int64

In [23]:
df['PAP'].value_counts(normalize=True)*100


1    60.0
0    40.0
Name: PAP, dtype: float64

60% participam de programa de alimentação, 40% não

In [241]:
df['GI'].value_counts()

3    23
2    11
1     6
Name: GI, dtype: int64

In [24]:
df['GI'].value_counts(normalize=True)*100

3    57.5
2    27.5
1    15.0
Name: GI, dtype: float64

57,5% das casas tem um chefe de familia com nenhum grau completo, 27.5% tem com só o primeiro grau completo, 15.0% com 2 grau completo.